icons/storybook.svg

```diff
@@ -1 +1,8 @@
-<svg stroke-miterlimit="1.4142" style="clip-rule:evenodd;fill-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:1.4142" version="1.1" viewBox="0 0 300 300" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><g transform="translate(11.5 2.2608)" style="fill:#ec407a"><g transform="translate(-.006806 .000978)"><path transform="translate(-11.5 -2.2608)" d="m257.53 22.271-18.676 1.377v36.545l-16.613-7.1191-16.613 7.1191v-34.096l-128.74 9.4922 11.061 234.02 169.59 8.1172zm-87.213 48.121v.002c31.663 0 49.853 17.289 48.955 52.32l-36.152.90039c.67367-18.863-7.4134-20.66-11.904-20.66-4.2666 0-11.451 1.3481-11.451 11.453 0 24.926 61.529 23.578 61.529 73.879 0 28.294-22.008 41.316-50.078 41.316-28.968 0-54.343-9.8807-51.424-53.221h37.951c-.44912 17.516 3.3698 23.807 13.025 23.807 7.4104 0 10.777-4.2674 10.777-11.453 0-25.6-60.629-26.501-60.629-73.434 0-26.947 17.738-44.91 49.4-44.91z" style="clip-rule:evenodd;fill-rule:nonzero;fill:#ec407a;paint-order:fill markers stroke"/><path d="m50.77 33.329h-19.795l8.2949 234.02h22.56z" style="clip-rule:evenodd;fill-rule:nonzero;fill:#ba68c8;paint-order:fill markers stroke"/></g></g></svg>
+<svg viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg">
+  <path
+    d="M10.14 41.86L8.9 8.84a2.01 2.01 0 011.88-2.08L38.86 5A2.01 2.01 0 0141 7.01V43a2.01 2.01 0 01-2.1 2l-26.85-1.2a2.01 2.01 0 01-1.91-1.93z"
+    fill="#FF4081" />
```

Comment:
Note that this is a change from `#ec407a` before. Storybook themselves use `#fe4785` and `#FF4081` was the closest match using https://pkief.github.io/material-color-converter/